src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/DigitalCaseFileService.java

```diff
@@ -106,5 +112,29 @@ private BundleDetails createBundleDetails(CaseData caseData) {
                 .documents(bundleDocuments)
                 .build();
     }
+
+    /**
+     * Prepare wordings to display digitalCaseFile link.
+     * @param caseData Get caseData
+     * @return Link with Markup
+     */
+    public String getReplyToReferralDCFLink(CaseData caseData) {
+        if (caseData.getDigitalCaseFile() != null) {
+            return formatReplyToReferralDCFLink(caseData.getDigitalCaseFile().getUploadedDocument());
+        }
+
+        return caseData.getDocumentCollection()
+            .stream()
+            .filter(d -> defaultIfEmpty(d.getValue().getTypeOfDocument(), "").equals(TRIBUNAL_CASE_FILE)
+                || defaultIfEmpty(d.getValue().getMiscDocuments(), "").equals(TRIBUNAL_CASE_FILE))
```

Comment:
Wouldn't "TRIBUNAL_CASE_FILE.equals(d.getValue().getMiscDocuments())" do the same thing without adding the extra import?